components/webdriver_server/actions.rs

```diff
@@ -95,20 +97,81 @@ fn compute_tick_duration(tick_actions: &ActionSequence) -> u64 {
 impl Handler {
     // https://w3c.github.io/webdriver/#dfn-dispatch-actions
     pub(crate) fn dispatch_actions(
-        &mut self,
+        &self,
         actions_by_tick: &[ActionSequence],
     ) -> Result<(), ErrorStatus> {
+        // Step 1. Wait for an action queue token with input state.
+        let new_token = self.id_generator.next();
+        self.current_action_id.set(Some(new_token));
```

Comment:
Is it possible to have nested waiting? Namely, dispatch an action while waiting for another one.

If not, we should assert that `current_action_id` is `None` here.